docs/generated/sql/bnf/show_transfer_stmt.bnf

```diff
@@ -0,0 +1,3 @@
+show_transfer_stmt ::=
+	'SHOW' 'TRANSFER' 'STATE' 'WITH' non_reserved_word_or_sconst
+	| 'SHOW' 'TRANSFER' 'STATE'
```

Comment:
Can we make the WITH clause required? The sql proxy will always supply the WITH clause and removing the option simplifies the implementation+tests.